Python/Product/Analysis/Interpreter/Ast/AstPythonMultipleMembers.cs

```diff
@@ -39,6 +39,13 @@ class AstPythonMultipleMembers : IPythonMultipleMembers, ILocatedMember {
             _checkForLazy = true;
         }
 
+        public IMember Trim() {
```

Comment:
Oh, calling `Members` might resize `_members` potentially going from size 1 to 0

Reply:
Good point though, should really check that here.

Reply:
Actually, decided not to change it. If we trim down to a single lazy member, we should probably leave it lazy even if it will eventually (depending on context) resolve down to nothing.